lib/network/modules/CanvasRenderer.js

```diff
@@ -166,9 +168,10 @@ class CanvasRenderer {
         }
       }
     } catch(e) {
-      console.warning("Got exception with message: '" + e.message() + "'");
-      if (e.message() === 'window is undefined') {
-        console.warning("'" + e.message() + "' happened again");
+      var msg = e.message;
+      //console.warn("Got exception with message: '" + msg + "'");
```

Comment:
Remove comment

Reply:
Ah, yes, that one. I hope the travis unit test thing is truly dead now.

Reply:
I'm getting rid of the catch-block, it's not required any more.